lightning/src/routing/network_graph.rs

```diff
@@ -409,7 +425,6 @@ where C::Target: chain::Access, L::Target: Logger
 		let first_blocknum = 0;
 		let number_of_blocks = 0xffffffff;
 		log_debug!(self.logger, "Sending query_channel_range peer={}, first_blocknum={}, number_of_blocks={}", log_pubkey!(their_node_id), first_blocknum, number_of_blocks);
```

Comment:
Would it be worth logging similarly for consistency?

Reply:
Its not conditional, so its not super informative, and it'll be logged at the TRACE level anyway cause we log all messages.